Nov release - apps (#4809)
SumoLogic/sumologic-documentation: blog-service/2024-11-28-apps.md

### New release

We’re excited to announce the release of the new Azure Database for PostgreSQL and Azure CosmosDB for Sumo Logic.

- **Azure Database for PostgreSQL**. Azure Database for PostgreSQL is a fully managed relational database service in the Microsoft cloud based on the PostgreSQL community edition. This integration helps in monitoring resource utilization and identifying slow queries to optimize your workloads and configure your server for the best performance.
[Learn more](/docs/integrations/microsoft-azure/azure-database-for-postgresql/).
- **Azure CosmosDB**. Azure Cosmos DB is a fully managed NoSQL and relational database for modern app development offering single-digit millisecond response times, automatic and instant scalability, along with guaranteed speed at any scale. This integration helps in monitoring the overall performance, failures, capacity, and operational health of all your Azure Cosmos DB resources.
[Learn more](/docs/integrations/microsoft-azure/azure-cosmos-db/).

### Enhancements 

We're excited to announce the release of the enhancements listed below for the Sumo Logic apps:

> - **Added Monitors in OpenTelemetry apps**. Sumologic provides six new monitors for Apache app with a number of alerts that are triggered on different conditions. [Learn more](/docs/integrations/web-servers/opentelemetry/apache-opentelemetry/#apache-alerts).

Reviewer: ```suggestion
- **Apache - OpenTelemetry**. Added six new monitors for Apache - OpenTelemetry app that will be triggered for different pre-defined conditions. [Learn more](/docs/integrations/web-servers/opentelemetry/apache-opentelemetry/#apache-alerts).
```

> - **Added Unified Audit log in OpenTelemetry apps**. Added the ability to gather Unified Audit logs using Syslog and Windowsevent receivers. Additionally, this dashboard offers information on database users, top current users, and trends in logon status. This dashboard can also be used with the unified audit logs exported from both Windows and Linux environments. [Learn more](/docs/integrations/databases/opentelemetry/oracle-opentelemetry/#unified-audit-syslog).

Reviewer: ```suggestion
- **Oracle - OpenTelemetry**.  Updated the collection process to fetch unified audit logs and added new **Unified Audit Syslog** dashboard. This new dashboard offers information on database users, top current users, and trends in logon status. This dashboard can also be used with the unified audit logs exported from both Windows and Linux environments. [Learn more](/docs/integrations/databases/opentelemetry/oracle-opentelemetry/#unified-audit-syslog).
```

> - **Added Unified Audit log in OpenTelemetry apps**. Added the ability to gather Unified Audit logs using Syslog and Windowsevent receivers. Additionally, this dashboard offers information on database users, top current users, and trends in logon status. This dashboard can also be used with the unified audit logs exported from both Windows and Linux environments. [Ldearn more](/docs/integrations/databases/opentelemetry/oracle-opentelemetry/#unified-audit-syslog).

Reviewer 2: ```suggestion
- **Added Unified Audit log in OpenTelemetry apps**. Added the ability to gather Unified Audit logs using Syslog and Windowsevent receivers. Additionally, this dashboard offers information on database users, top current users, and trends in logon status. This dashboard can also be used with the unified audit logs exported from both Windows and Linux environments. [Learn more](/docs/integrations/databases/opentelemetry/oracle-opentelemetry/#unified-audit-syslog).
```

> - **Added CloudTrail dashboard for AWS Application Load Balancer**. The CloudTrail Audit dashboard visualizes successful and failed events globally, event trends, error details, and user activities, offering insights into load balancer performance, security, and usage patterns. You can receive a comprehensive overview of the [AWS Application Load Balancer](/docs/integrations/amazon-aws/application-load-balancer/#cloudtrail-audit), [AWS Classic Load Balancer](/docs/integrations/amazon-aws/classic-load-balancer/#cloudtrail-audit), and [AWS Network Load Balancer](/docs/integrations/amazon-aws/network-load-balancer/#cloudtrail-audit), and their activities through the CloudTrail audit logs.

Reviewer: ```suggestion
- **Added CloudTrail Audit dashboard**. The CloudTrail Audit dashboard is added to the [AWS Application Load Balancer](/docs/integrations/amazon-aws/application-load-balancer/#cloudtrail-audit), [AWS Classic Load Balancer](/docs/integrations/amazon-aws/classic-load-balancer/#cloudtrail-audit), and [AWS Network Load Balancer](/docs/integrations/amazon-aws/network-load-balancer/#cloudtrail-audit) apps. This dashboard helps you to visualize the successful and failed events globally, event trends, error details, and user activities, offering insights into load balancer performance, security, and usage patterns. 
```

- **Amazon RDS**. Added dashboards of Oracle DB for CloudWatch Logs, CloudWatch Metrics, and CloudTrail logs. [Learn more](/docs/integrations/amazon-aws/rds/#oracle-logs---alert-logs-analysis).

### Source Template Enhancements

We're excited to announce the release of the enhancements listed below for the Source Templates in Sumo Logic apps:
- Update to 11 Source template (OTRM) with:

>     - log and metric section feature

Reviewer 2: ```suggestion
    - Log and metric section feature.
```
